src/linker/specifications/pipeline_2_0_0.yaml

```diff
@@ -1,5 +1,11 @@
 steps:
   step_1:
     implementation: step_1_python_pandas
+    # implementation_configuration:
```

Comment:
Should these be commented out? Also see comment above regarding this key name

Reply:
idk - I left them to help during development so we remember to turn them on/off for testing. They likely wouldn't be there in the finished product but would instead be documented as optional things

Reply:
Really `pipeline.yaml` in general won't be there in the finished product -- it will be provided by the user. So from an RT perspective I don't much care whether these are commented or not.

Reply:
Yeah, good point